### PR TITLE
Makes the code base unsafe

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Reference:

https://archive.is/LRWqP

### CODE REVIEW

**Comment by lzimann** [comment]
wut

**Comment by hornygranny** [comment]
:+1:

**Comment by optimumtact** [comment]
We already have a proc that does this :-1:

**Comment by DaedalusGame** [comment]
:100:

**Comment by PKPenguin321** [comment]
Meme

**Comment by MrPerson** [comment]
Memes, :-1: 

No need to be a dick about it

**Comment by clusterfack** [comment]
Nice meme

File: code/__DATASTRUCTURES/linked_lists.dm [hunk 1]
```diff
@@ -13,6 +13,8 @@
 	var/datum/linked_node/next_node = null
 	var/datum/linked_node/previous_node = null
 
+/datum/linked_list/proc/iGiveHead()
```
Comment (FlavoredCactus) [inline code/__DATASTRUCTURES/linked_lists.dm]: very lewd

**Comment by octareenroon91** [comment]
Please better implement the memes

**Comment by Shadowlight213** [comment]
:ok:

**Comment by Delimusca** [comment]
safe space prank ♂GONE SEXUAL♂

**Comment by optimumtact** [comment]
It's A SOCIAL EXPERIMENT BRO DON'T TASE ME LOOK THE CAMERAS

**Comment by Giacom** [comment]
On one hand, inconsistent code formatting, on the other, funny references.

**Comment by Supermichael777** [comment]
so some dev somewhere catches a ban at his local coder-base for trying to remove a mistake due to accidental email flood and the resulting vaguely power abusive STOPSTIRINGUPSHITGUYS response of banning them with no recourse.
24ish hours later we decide we need to make light of this by offending everyone.
Ayyyyyyyyyyyyy
at least we have some administrative standards even if how servers work makes us a monarchy

**Comment by Thunder12345** [comment]
Down with memes

**Comment by tkdrg** [comment]
it's funny but it's not getting merged guys

**Comment by optimumtact** [comment]
it's funny but it's not getting merged guys

**Pull request closed without merging**